Source/Charts/Data/Implementations/Standard/BarChartDataEntry.swift

```diff
@@ -102,17 +102,17 @@ open class BarChartDataEntry: ChartDataEntry
     
     open func sumBelow(stackIndex :Int) -> Double
     {
-        if _yVals == nil
+        guard let _yVals = _yVals else
```

Comment:
shall we keep using `_` for local instant here? seems a little bit weird, though we don't have to change other code.

Reply:
updated